### PR TITLE
Updated to support multi-slot transclusion

### PR DESCRIPTION
**Pull request by romiem**

### CODE REVIEW

**Comment by shyndman** [comment]
What about changes to the transclusion function?

`function([scope], cloneLinkingFn, futureParentElement, slotName)`:
...
`slotName:` (optional) the name of the slot to transclude. If falsy (e.g. null, undefined or '') then the default translusion is provided. The $transclude function also has a method on it, $transclude.isSlotFilled(slotName), which returns true if the specified slot contains content (i.e. one or more DOM nodes).

File: angularjs/angular.d.ts [hunk 1]
```diff
@@ -200,6 +200,7 @@ declare module angular {
          * @param inlineAnnotatedFunction Execute this function on module load. Useful for service configuration.
          */
         config(inlineAnnotatedFunction: any[]): IModule;
+        config(object: Object): IModule;
```
Comment (mhegazy) [inline angularjs/angular.d.ts]: Please do not use `Object`. use `any` instead.

File: angularjs/angular.d.ts [hunk 2]
```diff
@@ -1713,7 +1714,7 @@ declare module angular {
         /**
          * Whether transclusion is enabled. Enabled by default.
          */
-        transclude?: boolean;
+        transclude?: any;
```
Comment (mhegazy) [inline angularjs/angular.d.ts]: can you point to the documentation backing this change?

**Comment by mhegazy** [comment]
Thanks for the contribution. This PR contains merge conflicts and can not be merged at the moment. Please refresh this PR and address the merge conflicts.

File: angularjs/angular.d.ts (hunk 2 above)
Comment (romiem) [inline angularjs/angular.d.ts]: @mhegazy Here is a link to the docs backing this change: https://docs.angularjs.org/api/ng/directive/ngTransclude#multi-slot-transclusion
Look at the code sample.

**Comment by RyanCavanaugh** [comment]
Hello and thank you for your contribution!

Due to an excessively long queue of pull requests that have become stale, we are "declaring bankruptcy" and closing all PRs opened before May 1, 2016. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

Going forward, we are committing to review or merge all PRs on a regular basis so this bankruptcy will not occur again. We apologize for the incovenience and hope you will continue to contribute to DefinitelyTyped in the future.

Thanks
The TypeScript and DefinitelyTyped teams

**Pull request closed without merging**